code/modules/admin/player_panel.dm

```diff
@@ -86,8 +86,14 @@ var/datum/vueui_module/player_panel/global_player_panel
 			VUEUI_SET_CHECK(data["players"][ref]["antag"], special_char, ., data)
 		else
 			VUEUI_SET_CHECK(data["players"][ref]["antag"], -1, ., data)
-		if(isMod && M.client?.player_age)
-			var/age = M.client.player_age
+		if(isMod && (M.client?.player_age || M.player_age))
+			var/age = "Requires database"
+			if(M.client?.player_age)
+				age = M.client.player_age
+			else if(M.player_age)
+				age = M.player_age
+			else
+				age = "NA"
 			if(age == "Requires database")
```

Comment:
Isnt that redundant now?

Reply:
If game doesn't load age from database, `M.client.player_age` is set like that. So it would break ui a bit.